### PR TITLE
add docs for arm64

### LINKED ISSUE #28: Add support for ARM
- build multiarch images: https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
- update the update check mechanism to compare with the correct image of the image list
- build the images
- make sure that all containers work on both platforms
- test if updating/update check works from current builds
- peobably more

### PR DESCRIPTION
**Pull request by szaimen**
Close #28

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: develop.md [hunk 1]
```diff
@@ -14,40 +14,24 @@ nextcloud/all-in-one:develop
...
 
-## How to promote builds from develop to latest
-
-<!---
-You can use the Docker CLI to promote builds from develop to latest. Make sure to adjust:
+## How to build new containers
 
-- $name
-- $digest
+Go to https://github.com/nextcloud-releases/all-in-one/actions/workflows/repo-sync.yml and run the workflow that will first sync the repo and then build new container that automatically get published to `develop` and `develop-arm64`.
```
Comment (juliushaertl) [inline develop.md:19]: Just out of curiosity, is there a reason for the separate tags? It should be possible to build an image for multiple architectures using the same tag with https://docs.docker.com/buildx/working-with-buildx/

Comment (szaimen) [reply develop.md:19]: Yes, there is one :)

See https://github.com/nextcloud/all-in-one/pull/42#issuecomment-986057315

Comment (juliushaertl) [reply develop.md:19]: Ah I see, thanks for the quick reply :)

Comment (szaimen) [reply develop.md:19]: no problem! Thanks for your reviews! :)

**Pull request merged**